source/configuration_manual/namespace/index.rst

```diff
@@ -306,7 +306,11 @@ given list.
 ::
 
   userdb static {
-    args = namespace+=special namespace/special/mail_path=/var/special/%u namespace/special/prefix=special/
+    fields {
+        # TODO: Not sure if this is the right replacement for
+        # namespace+=special namespace/special/mail_path=/var/special/%u namespace/special/prefix=special/
```

Comment:
Yeah. Just add `namespace+ = special`. Hmm. That's kind of ugly. Maybe we should make it possible to use `namespace += special` so it automatically strips away the space before `+`

Reply:
Okay i removed the TODO and added `namespace+ = special`